internal/rpcserver/interface.go

```diff
@@ -456,3 +456,15 @@ type LogManager interface {
 	// is invalid.
 	ParseAndSetDebugLevels(debugLevel string) error
 }
+
+// SanityChecker represents a block sanity checker for use with the RPC server.
+// The purpose of this is to allow an alternative implementation to be used for
+// testing.
+//
+// The interface contract requires that all of these methods are safe for
+// concurrent access.
+type SanityChecker interface {
+	// CheckBlockSanity performs some preliminary checks on a block to ensure
```

Comment:
This comment doesn't make sense for the interface.  It needs to be framed by what is required of the implementer.